var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -287,6 +287,10 @@ def configure_args(self):
                 'FCFLAGS='  + self.compiler.fc_pic_flag,
             ])
 
+        # Fujitsu Compiler dose not add  Fortran runtime in rpath.
+        if self.spec.satisfies('%fj') and '+fortran' in self.spec:
```

Comment:
```suggestion
        if '+fortran %fj' in self.spec:
```
should be equivalent right? I think `__contains__` is just an alias to `satisfies`.

Reply:
Thank you @sethrj, this process worked fine. I accepted this suggestion.